javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,10 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  var justNames = [];
+  var i = students.length;
+  while (i > 0) {
+    justNames.unshift(students[i - 1].name);
+    i--;
+  }
```

Comment:
Same feedback here, there's no real reason this loop needs to be going backwards, please make it move forwards through the array.